README.md

```diff
@@ -1,285 +1,24 @@
-<a name="readme-top"></a>
```

Comment:
Please add a professional README as it's also requested by the previous code reviewer. You may use [this](https://github.com/microverseinc/readme-template) tempalate from Microverse.